extensions/core/runtime/src/main/java/org/apache/camel/quarkus/core/runtime/graal/SubstituteIntrospectionSupport.java

```diff
@@ -21,14 +21,17 @@
 import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.RecomputeFieldValue;
 import com.oracle.svm.core.annotate.TargetClass;
+import com.oracle.svm.core.annotate.TargetElement;
 import org.apache.camel.spi.BeanIntrospection;
 import org.apache.camel.support.LRUCacheFactory;
 
+//
+// Don't use class here to avoid warning triggered by IntrospectionSupport deprecation
+//
 @TargetClass(className = "org.apache.camel.support.IntrospectionSupport")
-final class Target_org_apache_camel_support_IntrospectionSupport {
-
+final class SubstituteIntrospectionSupport {
     @Alias
+    @TargetElement(name = "CACHE")
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.FromAlias)
-    private static Map<Class<?>, BeanIntrospection.ClassInfo> CACHE = LRUCacheFactory.newLRUWeakCache(256);
-
+    private static Map<Class<?>, BeanIntrospection.ClassInfo> cache = LRUCacheFactory.newLRUWeakCache(256);
```

Comment:
Why `@TargetElement(name = "CACHE")` and `cache` field? Is our checkstyle protesting against an uppercase field name given it is not final?

Reply:
Yes, one of the reason for the renaming was to be able to run checkstyle against substitutions but then `final` does not make much sense for substrate and uppercase makes checkstyle unhappy so this is a compromise.

As alternative I can explicit ignore that class.

Reply:
Thanks for the explanation. BTW, there is also `// CHECKSTYLE:OFF` and `// CHECKSTYLE:ON` that can be used to skip specific lines of code.

Reply:
Yes, everything adds a little bit of noise, if you feel the check-style comment is better, feel free to change it

Reply:
No, your solution is one line less :-)